Fix semver for prerelease versions

Kufat opened the pull request:
Noticed this issue while trying to install a plugin on 4.4.1-rc2.

```
> semver.default.satisfies("4.4.1-rc2", ">=4.3.0")
false
> semver.default.satisfies("4.4.1-rc2", ">=4.3.0", {includePrerelease: true})
true
```

brunnre8 commented:
please squash it down

brunnre8 commented:
Generally if it's something as trivial as this you can immediately rebase, there's no point in having this in the history

Kufat committed:
Fix semver for prerelease versions

Noticed this breakage while trying to install a plugin on 4.4.1-rc2.

```
> semver.default.satisfies("4.4.1-rc2", ">=4.3.0")
false
> semver.default.satisfies("4.4.1-rc2", ">=4.3.0", {includePrerelease: true})
true
```

brunnre8 commented:
Thank you
